src/app/core/service/post/post-mock.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { PostService } from './post.service';
+import { Post } from '../../model/Post';
+import { mockPosts } from './mockPosts';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class PostMockService implements PostService {
+    getPosts(): Post[] {
```

Comment:
Tu chyba powinniśmy podać id wydarzenia

Reply:
Te posty obecnie są tylko w celu wizualizacji. Zostawię to na razie dla wygody, później zajmę się refactorem podczas pracy nad ich wyglądem.